hooks/post/30-upgrade.rb

```diff
@@ -38,6 +38,15 @@ def import_subscriptions
   Kafo::Helpers.execute('foreman-rake katello:upgrades:2.4:import_subscriptions')
 end
 
+def remove_elasticsearch
+  gems = ['ruby193-rubygem-tire', 'tfm-rubygem-tire', 'elasticsearch', 'sigar', 'lucene3']
+  gems.each do |gem|
+    Kafo::Helpers.execute('rpm -e #{gem}')
```

Comment:
yum remove will remove the package you request and everything that depends on it, i would be hesitant to do that personally as it could remove something unintentional.